example/server/server.js

```diff
@@ -29,6 +29,10 @@ function openSocket() {
         interval = setInterval(function() {
           sendData();
         }, 500);
+
+        ws.on('error', (err) => {
+          console.log(err);
+        });
   });
 }
```

Comment:
Stops the server from crashing on browser refresh. Just annoying when you're developing.